src/main/webapp/app/exercises/programming/manage/grading/programming-exercise-configure-grading.component.html

```diff
@@ -205,6 +205,24 @@ <h2 class="mb-1 fw-medium" jhiTranslate="artemisApp.programmingExercise.configur
                                     {{ value }}
                                 </ng-template>
                             </ngx-datatable-column>
+                            <ngx-datatable-column name="Type" prop="programmingExerciseTestCaseType">
```

Comment:
This column should only be shown for programming exercise which support the distinction. I think this is only the case for Java at the moment (please double check)